cursorless-talon/src/spoken_scope_forms.py

```diff
@@ -0,0 +1,76 @@
+from talon import Context
+
+from .csv_overrides import csv_get_ctx, csv_get_normalized_ctx
+
+
+def init_scope_spoken_forms(graphemes_talon_list: dict[str, str]):
+    create_flattened_talon_list(csv_get_ctx(), graphemes_talon_list, True)
+    create_flattened_talon_list(csv_get_normalized_ctx(), graphemes_talon_list, False)
```

Comment:
If we can easily check if we are in the test mode (reading from scope), I would probably put this behind an if statement as a small step towards detangling that code from here